### ISSUE TITLE
TV4 - Can't find the video api / Cant find vid id for autonaming

### ISSUE THREAD

**Issue by Sopor-** [spaam/svtplay-dl#382]
I tried to download an episode but i get "Can't find the video api". I can watch the episode in my web browser. The episode seem to be removed now because it is not free anymore, but i was trying to download it before 23:59:59 and why do i get different error message?

Svtplay-dl 1.0
```
svtplay-dl.exe http://www.tv4play.se/program/brottsplats-sverige?video_id=3350494 -v
DEBUG [1461880965.4644876] svtplay_dl\utils\__init__.py/request: HTTP getting http://www.tv4play.se/program/brottsplats-sverige?video_id=3350494'
DEBUG [1461880965.542615] svtplay_dl\utils\__init__.py/request: HTTP getting http://premium.tv4play.se/api/web/asset/3350494/play'
ERROR [1461880965.5738897] svtplay_dl\__init__.py/get_one_media: Can't find the video api
```

If i instead run Svtplay-dl 1.0-27-g003c494 i get this message, Cant find vid id for autonaming
```
python svtplay-dl http://www.tv4play.se/program/brottsplats-sverige?video_id=3350494 -v
DEBUG [1461881522.38] svtplay-dl\svtplay_dl\utils\__init__.py/request: HTTP getting http://www.tv4play.se/program/brottsplats-sverige?video_id=3350494'
DEBUG [1461881522.57] svtplay-dl\svtplay_dl\utils\__init__.py/request: HTTP getting http://prima.tv4play.se/api/web/asset/3350494/play'
DEBUG [1461881522.78] svtplay-dl\svtplay_dl\utils\__init__.py/request: HTTP getting http://webapi.tv4play.se/play/video_assets?type=episode&is_live=false&platform=web&node_nids=brottsplats-sverige&per_page=99999'
DEBUG [1461881522.92] svtplay-dl\svtplay_dl\utils\__init__.py/request: HTTP getting http://webapi.tv4play.se/play/video_assets?type=clips&is_live=false&platform=web&node_nids=brottsplats-sverige&per_page=1000&page=1'
ERROR [1461881524.56] svtplay-dl\svtplay_dl\__init__.py/get_one_media: Cant find vid id for autonaming
```

**Comment by johannewman** [comment]
I have the same problem.

**Comment by ullgren** [comment]
Had the same problem. Switching to latest master fixed it for me.
However in the latest master the fix for  #336 has been reverted so I had to apply that fix again manually to get it working.

**Comment by spaam** [comment]
why use "http://www.tv4play.se/program/brottsplats-sverige?video_id=3350494" ? the acual video is not there :)

**Comment by Sopor-** [comment]
The video is now removed because of no more free watching, but it was there yesterday.

**Comment by carlholmberg** [comment]
Same thing happens with i.e. https://www.tv4play.se/program/maria-wern?video_id=3289819

**Comment by spaam** [comment]
@carlholmberg master have a fix for that

@Sopor- they changed the url for the video stream info. from premium to prima. i changed that after i made the 1.0 release.   why it happend with autonaming i have no idea.

**Comment by johannewman** [comment]
Stupid question but how do I install the latest master on Ubuntu?

Den sön 1 maj 2016 04:21Johan Andersson <notifications@github.com> skrev:

**Comment by Sopor-** [comment]
@johannewman you will find that in the [README](https://github.com/spaam/svtplay-dl/blob/master/README.rst)

**Comment by diskpart** [comment]
With latest master I'm guessing you mean Latest on the page https://svtplay-dl.se/archive?

I have tried the latest windows exe but still get the "Can't find the video api" when trying to download from TV4 Play.

svtplay-dl http://www.tv4play.se/program/gladiatorerna?video_id=3350284 -v
DEBUG [1462090652.5992467] svtplay_dl\utils\__init__.py/request: HTTP getting 'http://www.tv4play.se/program/gladiatorerna?video_id=3350284'
DEBUG [1462090652.7572558] svtplay_dl\utils\__init__.py/request: HTTP getting 'http://premium.tv4play.se/api/web/asset/3350284/play'
ERROR [1462090652.8392603] svtplay_dl\__init__.py/get_one_media: Can't find the video api

**Comment by qnorsten** [comment]
diskpart: with latest master I guess spaam means, the updated code from here. That you can get if you do a git clone of the master branch. Than you have to runt the python script manually, and make sure you have the required dependencys installed (like you had to do before he started distribution of exe file). 
If you do not wanna do that, I guess you will have to wait until spaam decides to release a new version. 
The exe file does not have the latest commits and bugfixes.

**Comment by spaam** [comment]
this should be fixed in the latest version 1.1.

**Comment by diskpart** [comment]
1.1 windows exe works perfectly!! Thanks for all the hard work! :+1:
